releasenotes/notes/cws-cmd-download-policy-0b19ce628afd289f.yaml

```
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The security agent now offers a command to directly download the policy file from the API.
+deprecations:
+  - |
+    The commands `check-policies` and `reload` of the security agent are deprecated.
+    Please use `runtime policy check` and `runtime policy reload` respectively instead.
```

```suggestion
    The security Agent commands `check-policies` and `reload` are deprecated.
    Use `runtime policy check` and `runtime policy reload` respectively instead.
```
Not sure, but these might require double back-ticks?

---

Good point, let me check in other release notes